mcs/class/Facades/System.Drawing.Common/TypeForwarders.cs

```diff
@@ -10,6 +10,9 @@
 // System.Drawing.Primitives/ref:
 namespace System.Drawing
 {
+
+#if !MONODROID
```

Comment:
Why do you need the ifdef if the file is not used for MONODROID

Reply:
Not sure I understand, this file is included. monodroid profiles sources include `System.Drawing.Common.dll.sources` which includes `TypeForwarders.cs`

Reply:
oh, I missed the #include